src/main/java/Main.java

```diff
@@ -0,0 +1,33 @@
+import static core.basesyntax.db.Storage.fruitData;
+
+import core.basesyntax.dataprocess.DataProcessor;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataReaderService;
+import core.basesyntax.service.DataWriterService;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.impl.DataReaderServiceImpl;
+import core.basesyntax.service.impl.DataWriterServiceImpl;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class Main {
+    private static final Path inputFilePath = Paths.get("src", "main", "resources", "input.csv");
+    private static final Path outputFilePath = Paths.get("src", "main", "resources", "output.csv");
+
+    public static void main(String[] args) {
+        DataReaderService dataReaderService = new DataReaderServiceImpl();
+        List<FruitTransaction> transactions = dataReaderService
+                .readDataFromFile(inputFilePath.toString());
```

Comment:
We should have two different services for reading file and parsing data to java objects: see my comment above